model/clusters_mgmt/v1/sts_type.model

```diff
@@ -49,4 +49,7 @@ struct STS {
 
 	// Secrets Manager ARN for the OIDC private key key.
 	OidcPrivateKeySecretArn String
+
+    // If true, cluster account and operator roles have managed policies attached
```

Comment:
missing `.` at the end of the comment, and looks like an indentation issue

Reply:
Fixed.